app/build.gradle

```diff
@@ -735,5 +738,7 @@ tasks.register("updateCookiesExtensionVersion", Copy) { task ->
     updateExtensionVersion(task, 'src/main/assets/extensions/cookies')
 }
 
-preBuild.dependsOn updateAdsExtensionVersion
-preBuild.dependsOn updateCookiesExtensionVersion
+preBuild.configure {
```

Comment:
What's `preBuild`? Does this run every time we execute a gradle command? Or is it smart and only executes before Android compilation jobs?

Reply:
I'm not sure where preBuild is set up. I tried to keep the logic consistent by hooking into `configure`, but we can also keep this line as is for now.